xinference/model/embedding/core.py

```diff
@@ -167,23 +171,27 @@ def cache(model_spec: EmbeddingModelSpec):
 
     from_modelscope: bool = model_spec.model_id.startswith("Xorbits/")
     if from_modelscope:
+        os.makedirs(XINFERENCE_DOWNLOAD_MS_MODEL_DIR, exist_ok=True)
```

Comment:
It changes the default behavior, maybe we need to specify it only when users set the environment.